INDEX

```diff
@@ -448,6 +448,16 @@
         "category": "content",
         "homepage": "http://qbittorrent.org/"
     },
+    "vault": {
+        "MANIFEST": "vault.json",
+        "name": "vault",
+        "icon": "https://icons.freenas.org/community-icons/vault.png",
+        "description": "Manage secrets and protect sensitive data.",
+        "official": false,
+        "primary_pkg": null,
```

Comment:
This looks good, but did you intend to set primary_pkg: null? It could probably be set to vault, so in the UI it shows the version of the freebsd vault package?

Reply:
An easy change! One moment